src/log/log.service.ts

```diff
@@ -112,7 +115,19 @@ export class LogService {
   }
 
   async getRefVolume(ref: string): Promise<any> {
-    return this.logRepository.getRefVolume(ref);
+    const logsWithoutEur = await this.logRepository.find({ where: { message: ref, fiat: Not(2) } });
```

Comment:
Besser auf den Fiat-Namen prüfen? Bin mir nicht sicher, ob die IDs in allen DBs gleich sind.

Reply:
In allen DBs ist die ID 2 Euro. Also passts.